### DISCUSSION TITLE
Presenting other scripts (e.g., Arabic)

### DISCUSSION THREAD

**Discussion by lauderui** [jspsych/jsPsych#813] [Q&A] [answered]
Is it possible to present other scripts in an experiment? I have in the past worked with PsychoPy and Pavlovia, but there were issues with using Arabic. Specifically, the line wrap/size adjustment to different screens didn't work. Does jsPsych support scripts like Chinese, Japanese, Arabic? And if yes, how would that be done?

**Comment by jodeleeuw** [comment, marked as answer]
Hi @lauderui,

This should be possible. I know that there is a sizable contingent of Japanese researchers using jsPsych. I believe the key is to use the appropriate `<meta>` tags in the `<head>` section of the HTML file to let the browser know how the text is encoded. For example, if you using Arabic you may need to use a `<meta>` tag that specifies the document character set is `UTF-8` encoded:

```html
<meta http-equiv='content-Type' content='text/html; charset=UTF-8'/>
```

I don't have firsthand experience with this, so please let me know if you run into any issues with a different script and we'll figure out what's necessary. This would be a great addition to our documentation.

**Reply by jkhartshorne** [reply]
@lauderui my recollection is that when we've had to do this, all we needed to was to make sure the charset was UTF-8, as described by  @jodeleeuw. If you try that and it doesn't work, please let us know!
